src/game/gameOverModal.scss

```diff
@@ -0,0 +1,61 @@
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: $white-color;
+  &::before {
+    position: absolute;
+    content: "";
+    top: 0;
+    left: 15px;
+    right: 15px;
+    bottom: 0;
+    background-color: $black-dark-color;
+    @media screen and (min-width: $m-breakpoint) {
+      left: 45px;
+      right: 45px;
+    }
+  }
+  &__content {
+    max-width: 80vw;
+    max-height: 80vh;
+    z-index: 999;
+    h2 {
```

Comment:
I don't remember if I mentioned it here or not but NEVER don't style by `tags`. Only if it is required or the situation is good for it. Same thing for `h3`, and `a`.

Reply:
I have add classes and styled using them